### PR TITLE
docs: skeleton of developer docs

### PR DESCRIPTION
**Pull request by hongriTianqi**
A newly proposed skeleton of developer documentation.

### CODE REVIEW

**Comment by caic99** [comment]
Hi @hongriTianqi ,
I suggest keeping `contributing.md` in the original space. In [the contribute page](https://github.com/deepmodeling/abacus-develop/contribute), this file will be presented. [ref](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file)

**Comment by hongriTianqi** [comment]
> Hi @hongriTianqi , I suggest keeping `contributing.md` in the original space. In [the contribute page](https://github.com/deepmodeling/abacus-develop/contribute), this file will be presented. [ref](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file)

OK, done.

File: docs/community/contribution_guide.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ABACUS Contribution Guide
+
+## Contribution Process
+
+We are welcoming open source contributions from the community. The technical guide is provided in [Contributing to ABACUS](../CONTRIBUTING.md). Here is the basic contribution process:
```
Comment (caic99) [inline docs/community/contribution_guide.md]:
```suggestion
We welcome contributions from the open source community. The technical guide is provided in [Contributing to ABACUS](../CONTRIBUTING.md). Here is the basic contribution process:
```

**Review by caic99 — APPROVED** [review]
LGTM! Let's see what other people said.

**Review by dyzheng — APPROVED** [review]

**Pull request merged**